Rovers/Rovers.tex

```diff
@@ -0,0 +1,405 @@
+Four of those include rovers. 
+To ignore this fact would be to ignore a large part of space exploration.
+Currently RIT SPEX is not involved with rovers.
+It would be beneficial to our members to get some experience in this area.
+
+\subsection{Promotions and Funding}
+\label{subsec:promos}
+A rover would be a popular demonstration at ImagineRIT.
+The rover would be rather large and would attract many eyes. 
+We could even demo it outside if there is sufficient space. 
+A rover would be very easy to get video and photos of for SPEX promotions. 
+Having a rover is also another opportunity for SPEX to fundraise. 
+There is plenty of space to place company logos on the body of the rover. 
+It would also allow for SPEX to reach out to robotics companies.
+
+\subsection{Computer Science}
+\label{subsec:cs}
+\subsubsection{Member Retention and Expansion}
+\label{subsubsec:cs}
+The potential self-driving component would be the heaviest computer science project SPEX would have attempted, this would help with retention of CS and SE majors. 
+There are many features included in the MVP that require significant computer science skills as well.
+This includes navigation, power management, communications and more.
+That will give computing-related majors another option for SPEX involvement, also helping retention.
+
+\subsubsection{Artificial Intelligence}
+\label{subsubsec:ai}
+Some of the potential self-driving features will likely be implemented with artificial intelligence. In particular the fully autonomous navigation would require this.
+Machine learning and artificial intelligence are at the forefront of computer science right now. 
+They are heavily desired in industry, including space exploration.
+There are limited options for RIT students to get involved with AI project wise, this can help with that. 
+There is another AI-based student group (RITficial Intelligence) on campus that has members interested in these projects but the group lacks the funding to start any.
+RIT SPEX could partner with this group to recruit talent.
+
+\subsection{Robotics}
+\label{subsec:robotics}
+Robotics are also a heavily desired area in the space industry. 
+Currently RIT SPEX does not have any projects with robotics as a focus. 
+Robotics is also a potential area for collaboration with faculty as there are RIT faculty that are investigating this area.
+RIT is host to the FIRST robotics competition which would be a good source to get information and help from. 
+However, most university students volunteer to run this competition instead of participating. 
+This competition is at the high school level and many students want an opportunity to do something more advanced. 
+SPEX could even show off the rover at the FIRST robotics competition (especially if the rover competed in the University Robotics Competition or a similar competition). 
+RIT would love the promotional value an it would be a great recruitment and fundraising opportunity for RIT SPEX.
```

Comment:
How would FIRST robotics competition mentorship act as a fundraising opportunity for SPEX?

Reply:
RIT could pay us and we can raise funds via our donation portal